From ab32dbc24373364825fa8fba2e1a4ee6acc335a7 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Fri, 29 Jun 2018 09:54:29 +0800
Subject: [PATCH 040/205] arm: omap2: include OMAP_GPMC and MEMORY

This patch is to enable OMAP_GPMC by default since it's only bool.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/mach-omap2/Kconfig |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-omap2/Kconfig b/arch/arm/mach-omap2/Kconfig
index 214a516..f1717b3 100644
--- a/arch/arm/mach-omap2/Kconfig
+++ b/arch/arm/mach-omap2/Kconfig
@@ -57,6 +57,8 @@ config SOC_AM33XX
 	bool "TI AM33XX"
 	depends on ARCH_MULTI_V7
 	select ARCH_OMAP2PLUS
+	select MEMORY
+	select OMAP_GPMC
 	select ARM_CPU_SUSPEND if PM
 
 config SOC_AM43XX
-- 
1.7.9.5

